README.md

```diff
@@ -39,7 +39,20 @@ npm install openapi-factory
 ```javascript
 const Api = require('openapi-factory');
 const { Authorizer, RequestLogger } = require('microservice-utilities');
-const configuration = { jwkKeyListUrl: 'https://example.com/.well-known/jwks.json' };
+const configuration = {
+  // URL to get jwk keys to verify token
+  jwkKeyListUrl: 'https://example.com/.well-known/jwks.json',
+  // custom resolver that returns additional context form the AWS Lambda authorizer
+  authorizerContextResolver: () => ({
+    authorizerContextResult: 'my context'
+  }),
+  // optional usage plan that's being created
```

Comment:
Perhaps say `API Gateway usage plan` and link here: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-api-usage-plans.html